src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -6314,6 +6312,54 @@ void M()
             Assert.Equal("this, x", GetSymbolNamesJoined(dataFlowAnalysisResults.WrittenOutside));
         }
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/30548")]
```

Comment:
why skip?

Reply:
because I wasn't sure what is expected behavior. either "test" shouldn't be in data flow in or "test" should be in read inside.

test is added just to reduce labor for compiler team.

Reply:
It would be better ot be not skipped.  If the test is correct, there's nothing that needs to chane.  if the test is incorrect, it's just docmenting existing behavior and will break if the compiler thinks that behavior needs to change.  otherwise, it's just another skipped test sitting around that never gets run or looked at again.

Reply:
I have added tests for compiler team this way many times. never saw they left it as skipped once they fixed the bug.